rfcs/2021-03-02-6600-vector-remap-language-web.md

```diff
@@ -0,0 +1,169 @@
+# RFC 6600 - 2021-03-03 - Vector Remap Language on the web
+
+From the beginning of our work on [Vector Remap Language][VRL] (VRL), we've had discussions about
+novel ways to make VRL approachable and even downright fun for users. Thus far, we've done some
+pretty unique things, like providing a REPL for the language. But we've also had plans, though
+fairly unspecific ones, about making VRL available on the web in some form. This RFC seeks to lend
+some concreteness to those discussions.
+
+## Options
+
+In this RFC, I discuss three possibities. The first two emerged from our internal discussions, while
+the third was gathered from a survey of the Wasm ecosystem.
+
+1. An in-browser [web app][web_app]/playground for experimenting with and learning VRL
+2. A [web server][web_server] that can process VRL programs and events
+3. A [published Wasm module], largely for internal use
+
+I propose pursuing #1 and #3 in the near term, while leaving #2 as an option for the future.
+
+### Interactive web app
+
+An interactive VRL web app would be a browser-based app that provides an aesthetically pleasing and
```

Comment:
Just want to mention that I think the design of this is related to the upcoming work on the docs site.  We are looking to choose an open source library to replace the theming and components that was coming from TUI.  Whatever we land on there, I think it's going to make sense to use it here as well.

Reply:
That makes sense. Let's sync up about this soon, as we'll need to get people from the DD site involved as well, as they have their own imperatives that we'll need to heed.